config/settings.yml

```diff
@@ -68,6 +68,7 @@ encode_throttling:
   update_jobs_throttle_threshold: 3
   update_jobs_spacing: 10
 auth:
+  registerable: true
```

Comment:
I agree it is kind of a toss up.  I think long term on by default means folks spinning up a test instance can more easily get it going, but that it will need to be clearly mentioned in the release notes.